Makefile

```diff
@@ -30,14 +79,17 @@ version:
 	@echo $(VERSION)
 
 ./bin/node-problem-detector: $(PKG_SOURCES)
-	GOOS=linux go build -o bin/node-problem-detector \
+	CGO_ENABLED=$(CGO_ENABLED) GOOS=linux go build -o bin/node-problem-detector \
 	     -ldflags '-w -extldflags "-static" -X $(PKG)/pkg/version.version=$(VERSION)' \
```

Comment:
@ankon 

Yes, you're right. `-static` should only be used when journald is disabled. For more info please consult the comments about [this review](https://github.com/kubernetes/node-problem-detector/pull/54#pullrequestreview-14090197)